src/core/src/UsdNotificationHandler.cpp

```diff
@@ -397,6 +398,39 @@ void processCesiumPolygonRasterOverlayChanged(
     }
 }
 
+void processCesiumWebMapServiceRasterOverlayChanged(
+    const Context& context,
+    const pxr::SdfPath& webMapServiceRasterOverlayPath,
+    const std::vector<pxr::TfToken>& properties) {
+
+    const auto pWebMapServiceRasterOverlay =
+        context.getAssetRegistry().getWebMapServiceRasterOverlay(webMapServiceRasterOverlayPath);
+    if (!pWebMapServiceRasterOverlay) {
+        return;
+    }
+
+    // Process base class first
+    processCesiumRasterOverlayChanged(context, webMapServiceRasterOverlayPath, properties);
+
+    auto reload = false;
+    auto updateBindings = false;
+
+    for (const auto& property : properties) {
+        if (property == pxr::CesiumTokens->cesiumBaseUrl || property == pxr::CesiumTokens->cesiumLayers) {
```

Comment:
Should this also be checking `tileWidth`, `tileHeight`, `minimumLevel`, `maximumLevel`?

Reply:
I'm hoping to get @r-veenstra's feedback on this one. With all the potential USD notifications from sliders, I'm concerned about spamming and too many reloadings. I'm wondering if the solution is to add a "Refresh" button or something to the WmsRasterOverlay.

Reply:
A refresh button wouldn't be a bad idea, or ensuring Refresh Tileset button on the tileset also updates the overlays

Reply:
@corybarr I think we should still make this change in `UsdNotificationHandler` since the UI debounce issue is a broader issue.

Reply:
Sounds good. Let's go ahead and do that in this PR. I think there's some UX/UI we need to take a detailed look at before the next release. I created https://github.com/CesiumGS/cesium-omniverse/issues/676 to get it on the board.

Reply:
@lilleyse I added the check for these four properties.

Tangentially, the genericness of these tokens (`pxr::CesiumTokens->cesiumLayers`, `pxr::CesiumTokens->cesiumMinimumLevel`, etc) make me nervous. Would token-name collisions ever be an issue?

Reply:
> Would token-name collisions ever be an issue?

I don't think so since it's context dependent. Maybe it would be a problem if a base class and sub class had the same token but I doubt the schema code generator would allow that.